src/status_im/chat/models/input.cljs

```diff
@@ -134,6 +134,33 @@
              (update-in [:chat/inputs current-chat-id :metadata]
                         dissoc :sending-image))}))
 
+(fx/defn soft-delete-message-success
+  {:events [::soft-delete-message-success]}
+  [{:keys [db]} chat-id message-id]
+  (let [new-last-message (-> db
```

Comment:
why doesn't `wakuext_deleteMessageAndSend` return updated chats with updated last message ?

Reply:
i mean it doesn't look correct, to set last message in status-react

Reply:
https://github.com/status-im/status-react/pull/12390#discussion_r683900075